CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+## neptune-tensorflow-keras 2.0.0
+
+### Changes
+- `NeptuneCallback` now accepts namespace `Handler` as an alternative to `Run` for the `run` argument. This means that
+  you can call it like `NeptuneCallback(run=run["some/namespace/"])` to log everything to the `some/namespace/` 
+  location of the run.
+
+### Breaking changes
+- We removed support for the legacy Neptune client API. Use it with `import neptune.new as neptune`.
```

Comment:
```suggestion
- We removed support for the legacy Neptune client API. Use it with `import neptune.legacy as neptune`.
```

Reply:
The previous one was correct.

Reply:
Either I misunderstand or there must be something off with the phrasing. How can one use the legacy API by importing `neptune.new`?